tools/mo/unit_tests/mo/moc_frontend/moc_extractor_test_actual.py

```diff
@@ -84,8 +83,8 @@ def test_decode_name_with_port_delim_op_out(self):
         model_stat = get_model_statistic()
         place_stat = get_place_statistic()
 
-        assert model_stat.get_place_by_tensor_name == 1
-        assert model_stat.get_place_by_operation_name == 1
+        assert model_stat.get_place_by_tensor_name == 2
```

Comment:
Getting 2 times 'tensor name' and 3 times 'operation name' just for 'operation:7' looks too much. If number of calls can't be reduced, I'd suggest to add some comment about why so much calls are expected

Reply:
I was able to reduce the number of function calls